common/djangoapps/student/views.py

```diff
@@ -2486,18 +2486,22 @@ def validate_password(user, password):
         password: the user's proposed new password.
 
     Returns:
-        is_valid_password: a boolean indicating if the new password
-            passes the validation.
+        is_security_violated: a boolean indicating if the new password
```

Comment:
I think the variable name `is_security_violated` contradicted with the function name. The function name is to `validate_password`. I would expect the return to be `True` when password is valid, and `False` when password is invalid. When you update the return this way, you made the return value to be opposite of the convention.